contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@gierschv I have no idea if that would work (am not a license expert).
Otherwise, one can probably rewrite the same thing from scratch, but...
Maybe Loic can provide input on what should be done?